app/views/users/show.html.slim

```diff
@@ -0,0 +1,12 @@
+p#notice = notice
+
+p
+  strong Email:
+  = @user.email
+p
+  strong Password:
+  = @user.password
+
+=> link_to 'Edit', edit_user_path(@user)
```

Comment:
do we really need the show page at all? could we not use use edit? also showing user passwords again?